build/js/live-editor.output_pjs.js

```diff
@@ -1221,6 +1342,11 @@ window.PJSOutput = Backbone.View.extend({
         this.bind();
  
         this.build(this.$canvas[0]);
+        
+        this.resourceCache = new PJSResourceCache({
+            canvas: this.canvas,
+            output: this.output
```

Comment:
The reason why I'm passing the whole ```output``` object instead of just ```imagesDir``` and ```soundsDir``` is because ```setPaths``` is called asynchronously on the first run so we don't actually know the value for those paths yet.